### PR TITLE
Basic 404 page

### LINKED ISSUE #336: Create 404 page

### PR DESCRIPTION
**Pull request by a-ch-chu**
Fulfils issue #336.

### CODE REVIEW

**Comment by a-ch-chu** [comment]
Graphic based off [this image](https://openclipart.org/detail/11252/couch) from OpenClipart. A vector version would probably be preferable.

**Comment by aapeliv** [comment]
> Graphic based off [this image](https://openclipart.org/detail/11252/couch) from OpenClipart. A vector version would probably be preferable.

Thanks! For posterity: Open Clipart says this is licensed under CC0 (public domain).

**Comment by lucaslcode** [comment]
Is this still a WIP? Also next time you can make a branch in our repo, that way the CI will work and we can preview without checking out the branch.

File: app/frontend/src/AppRoutes.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ import Jail from "./features/auth/jail/Jail";
 import TOS from "./components/TOS";
 import { useAuthContext } from "./features/auth/AuthProvider";
 import { ConnectionsPage } from "./features/connections";
+import NotFound from "./features/NotFoundPage";
```
Comment (lucaslcode) [inline app/frontend/src/AppRoutes.tsx]:
```suggestion
import NotFoundPage from "./features/NotFoundPage";
```

File: app/frontend/src/AppRoutes.tsx [hunk 2]
```diff
@@ -81,6 +83,10 @@ export default function AppRoutes() {
       <PrivateRoute exact path="/">
         <Home />
       </PrivateRoute>
+      <Route exact path={notFoundRoute}>
+        <NotFound />
```
Comment (lucaslcode) [inline app/frontend/src/AppRoutes.tsx]:
```suggestion
        <NotFoundPage />
```

File: app/frontend/src/features/NotFoundPage.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    transform: "translate(-50%,-50%)",
+    textAlign: "center",
+  },
+  graphic: {
+    height: "75%",
+    width: "75%",
+  },
+})
+
+export default function NotFound() {
```
Comment (lucaslcode) [inline app/frontend/src/features/NotFoundPage.tsx]:
```suggestion
export default function NotFoundPage() {
```

**Review by lucaslcode — CHANGES_REQUESTED** [review]
I'm going to be annoying and say we should make everything *Page even though there are a few older ones which aren't. Otherwise ready to merge!

**Review by lucaslcode — DISMISSED** [review]

**Review by lucaslcode — APPROVED** [review]

**Pull request merged**